src/token/ERC20.sol

```diff
@@ -131,7 +131,7 @@ contract ERC20 is Auth, IERC20Metadata, IERC20Permit {
         emit Transfer(address(0), to, value);
     }
 
-    function burn(address from, uint256 value) external auth {
+    function burn(address from, uint256 value) public auth {
```

Comment:
Any particular reason why this is changed to public ?

Reply:
No, good catch. Was to be able to wrap it in the `SimpleERC20Wrapper`, but ended up going another way. Will revert